### PR TITLE
Avoid conflict when copying symbolic links

### PR DESCRIPTION
**Pull request by TrumanLing**
Fix bug #142: avoid symbolic link conflict.

### CODE REVIEW

**Comment by lucasdemarchi** [comment]
LGTM, thanks!

**Comment by lucasdemarchi** [comment]
Thinking again, there's one corner case. If previously `dest_path` was a directory and is now a link, the `os.remove()` call will fail and raise an exception. So you may want to try copying and handle the cases in which it fails, or handle the exception on the `os.remove()`

**Comment by TrumanLing** [comment]
@lucasdemarchi I have updated my modification via "ls -nfs", which is your original idea, I think.

File: mkosi [hunk 1]
```diff
@@ -1436,7 +1436,10 @@ def copy_git_files(src, dest, *, git_files):
         directory = os.path.dirname(dest_path)
         os.makedirs(directory, exist_ok=True)
 
-        shutil.copy2(src_path, dest_path, follow_symlinks=False)
+        if os.path.islink(src_path):
+            subprocess.run(['ls','-nfs', src_path, dest_path])
```
Comment (keszybz) [inline mkosi]: `ln` not `ls`!

Also a space is missing before `'-nfs'`.

**Comment by TrumanLing** [comment]
@keszybz I am sorry for my handwriting mistake, and I have updated it.

**Comment by keszybz** [comment]
#183 implements copying internally instead. Let's close this. I'll add a comment in the other PR why.

**Pull request closed without merging**